schema/iea43_wra_data_model.schema.json

```diff
@@ -744,6 +744,15 @@
                     "quality",
                     "carrier_to_noise_ratio",
                     "doppler_spectral_broadening",
+                    "true_north_offset",
+                    "orientation",
+                    "tilt_x",
+                    "tilt_y",
+                    "tilt_z",
+                    "elevation",
+                    "altitude",
+                    "azimuth",
```

Comment:
How azimuth relates to orientation and compass_direction? We may be using different tags to the same definition. Should we extend this with examples? Perhaps a point of discussion for our next meeting.

Reply:
> How azimuth relates to orientation and compass_direction?

Azimuth and altitude used together make up a celestial coordinate system. In our case it would be defining the position of the sun. Even though Azimuth is effectively an orientation/compass direction i would be inclined to keep it as it is commonly used in solar.

> Should we extend this with examples? 

Yeah, maybe we should put a definition in with examples. I'll look into that.

> Perhaps a point of discussion for our next meeting.

We are going to have to do this remotely as the next meeting will hopefully be finalizing the launch stuff.

Reply:
For me it is clear now. An example would be nice but I don't see as a requirement.

Reply:
@stephenholleran @kersting I agree with updating documentation. That way the spec / documentation / sql statements can always to stay in sync.  I think the keywords in themselves are not specific enough to be used if I am someone starting out fresh.